cmd/skopeo/sync.go

```diff
@@ -368,61 +378,158 @@ func imagesToCopyFromRegistry(registryName string, cfg registrySyncConfig, sourc
 			Context:   serverCtx})
 	}
 
-	for imageName, tagRegex := range cfg.ImagesByTagRegex {
-		repoLogger := logrus.WithFields(logrus.Fields{
+	// include repository descriptors for cfg.ImagesByTagRegex
+	{
```

Comment:
Non-blocking: I don’t think the `{ }` blocks are that necessary — and the comments don’t say anything new either.

(Absolutely non-blocking: Possibly `s/filterCollection/filters/g`, `s/additionRepoDescList/descs/`; the variables have short enough scope that short names are sufficient. But this works just fine.)